### ISSUE TITLE
Link to desktop won't create properly

### ISSUE THREAD

**Issue by C4Framework** [LinusU/node-appdmg#160]
Hi, when I use the following: 

```
{ "x": 480, "y": 110, "type": "link", "path": "/Desktop" },
```

The asset included is a white document alias, rather than a folder, and dragging a file over top of the asset doesn't work.

Using ```/Applications``` works! But, as part of my .dmg I would like people to drop some files onto the desktop like so:

<img width="752" alt="screen shot 2018-02-09 at 10 22 12 am" src="https://user-images.githubusercontent.com/1447498/36043168-3951bfcc-0d83-11e8-93a5-d06e3f51b84f.png">

**Comment by LinusU** [comment]
Unfortunately for you, the desktop doesn't reside at `/Desktop`, but rather `/Users/<username>/Desktop`

I don't think it's possible to get a link to the desktop working since it will have a different username for each user...

**Comment by LinusU** [comment]
Closing because of inactivity, feel free to try `~/Desktop` and report back ☺️

**Comment by Powersawer** [comment]
Hello! I am having the same issue. I was just about to accept that creating a usable universal desktop alias is not possible, but it can be done with apples hdutil pretty easily. I attached a sample DMG file, I tested it on my coworkers' machines and they were able to use the desktop alias.
This topic is old, so I was wondering if maybe there is some way to incorporate this into the appdmg workflow.

The File: https://easyupload.io/d94fvu

If that File is too sketchy for you, what I did was simply create a fresh Image with the HD Utility GUI and then dragged an alias of the desktop into it. After sending it do different people, they were able to utilize the alias and drag the included sample file to their desktop.

Kind regards
